lib/views/feedbacks_slack_view.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Aprb.Views.FeedbacksSlackView do
+
+  def render(event) do
+    %{
+      text: "#{event["subject"]["display"]} #{event["verb"]} #{event["properties"]["message"]} from #{event["properties"]["url"]}",
```

Comment:
Oh good to know. 

We couldn't figure out getting local APRb to talk to staging Rabbit, so maybe we can try now that all the PRs have been merged to staging.